cves/kernel/CVE-2019-12378.yml

```diff
@@ -280,8 +279,8 @@ ipc:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: false
+  note: The function is allocating memory within the kernel using kmalloc, this is not interprocess communication
```

Comment:
Why is memory allocation not an interprocess communication (just for more context, maybe).

Reply:
updated